### DISCUSSION TITLE
Illegal mix of collations - Installing filament/notification on a Previous filament/filament (adminpanel)

### DISCUSSION THREAD

**Discussion by ehs4nm** [filamentphp/filament#4114] [Help]
Trying to add notification (database) to the project. after installing and compiling assets this error comes up.

```SQLSTATE[HY000]: General error: 1267 Illegal mix of collations (utf8_general_ci,COERCIBLE) and (utf8_unicode_ci,COERCIBLE) for operation '=' (SQL: select count(*) as aggregate from `notifications` where `notifications`.`notifiable_type` = App\Models\User and `notifications`.`notifiable_id` = 1 and `notifications`.`notifiable_id` is not null and json_unquote(json_extract(`data`, '$."format"')) = filament and `read_at` is null) (View: .\vendor\filament\notifications\resources\views\components\database\index.blade.php) ```

As i trace the problem to the Notifications class in "Filament\Notifications\Http\Livewire" and getDatabaseNotificationsQuery() method which query like this:
return $this->getUser()->notifications()->where('data->format', 'filament');

the "->where('data->format', 'filament')" part is making the problem.

my database collation is (utf8_unicode_ci)

where does this collation conflicts comes from. I've never used utf8_general_ci in my project anywhere. what should i do to fix it?
Thanks, by the way, until i find an stable solution, how can i remove that part of query? (overriding that package method)

**Comment by danharrin** [comment]
If `where('data->format', 'filament')` is not supported in your database, surely this is a Laravel issue? This is standard Eloquent query syntax
